ros_buildfarm/templates/snippet/builder_system-groovy_check-for-failing-jobs.xml.em

```diff
@@ -0,0 +1,82 @@
+@(SNIPPET(
+  'builder_system-groovy',
+  command=
+"""// CHECK FOR FAILING JOBS
+import hudson.model.Job
+
+// Control the number of failures in a row to check
+min_number_of_failures = 5
+// Control the minimun amount of time failing
+min_days_failing = 7
+// rosdistro to search
+rosdistro = "@ros_distro"
+
+boolean is_failing_from_n_days(
+  int min_days_failing, Job job)
+{
+  now = new Date()
+  start_of_failing_period = now - min_days_failing
+
+  last_success_build = job.getLastSuccessfulBuild()
+  
+  // if there is a last success build check when it was  
+  if (last_success_build) {
+    return last_success_build.getTime() < start_of_failing_period
+  }
+  
+  // all builds are failing, check when was the first one
+  return job.getFirstBuild().getTime() < start_of_failing_period       
+}
+
+
+// check if the N previous from referece_build_num failed
```

Comment:
Spelling: `referece_build_num`

Also what is `referece_build_num` trying to reference? I couldn't find `reference_build_num` anywhere.